docs/index.md

```diff
@@ -1175,6 +1175,15 @@ Mocha allows you to define custom reporters. For more information see the [wiki]
 
 **The HTML reporter is not intended for use on the command-line.**
 
+### Printing error messages immediately
```

Comment:
Thanks for updating the docs!

Reply:
Think this sentence reads better without the word "immediately".

| The `spec`, `dot`, `list`, and `min` reporters support a `showErrorsImmediately` option that will output error details when they occur instead of after all tests have finished: